examples/fyne/layout.rye

```diff
@@ -0,0 +1,34 @@
+
+rye .needs { fyne }
+
+do\in fyne {
+
+    box: container-grid-cols 2 [
```

Comment:
I could get the first param (2) from the amount of block items passed. But since fyne could also do this but still left the number of cols/rows as param, I decided to do the same. What do you think?

Reply:
I think the good default is to stick to Go's api, unless we (maybe later when using) see that something could or should really be shortened as it's there for just small percentage of cases. 

One option would be that grid-cols wouldn't accept a number and grid-cols\n would, but for now I would leave a default, unless you already see, when doing the examples that the code is much better flowing without the number. Then add grid-cols\n option maybe.